packages/odyssey-react-mui/src/Banner.tsx

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { AlertColor } from "@mui/material";
+import React, { SyntheticEvent } from "react";
+import { Alert, Link } from "./";
+
+export interface BannerProps {
+  /**
+   * Determine the color and icon of the alert
+   */
+  severity?: AlertColor;
+  /**
+   * Sets the ARIA role of the alert
+   * ("status" for something that dynamically updates, null for something
+   * unchanging)
+   */
+  role?: "status" | undefined;
+  /**
+   * The text content of the alert
+   */
+  children: string;
+  /**
+   * The function that's fired when the user clicks the close button. If undefined,
+   * the close button will not be shown.
+   */
+  onClose?: (event: SyntheticEvent<HTMLButtonElement>) => void;
```

Comment:
This is one of the complaints from CI. Might be an overly specific typing? 

@KevinGhadyani-Okta what's appropriate here?

Reply:
There's a mouse event handler type we could use. I have to look it up to be sure, but there are actual types you can use rather than defining the event yourself.

The type itself looks fine, but we're also missing the `() => void` type too. The handler type does that for you.